tests/printing/transformed/lazy-vals-new.check

```diff
@@ -0,0 +1,106 @@
+[[syntax trees at end of MegaPhase{dropOuterAccessors, checkNoSuperThis, flatten, transformWildcards, moveStatic, expandPrivate, restoreScopes, selectStatic, Collect entry points, collectSuperCalls, repeatableAnnotations}]] // tests/printing/transformed/lazy-vals-new.scala
+package <empty> {
+  @SourceFile("tests/printing/transformed/lazy-vals-new.scala") final module 
+    class
+   A extends Object {
+    def <init>(): Unit = 
+      {
+        super()
+        ()
+      }
+    @static private def <clinit>(): Unit = 
+      {
+        A.OFFSET$_m_0 = 
+          scala.runtime.LazyVals.getStaticFieldOffset(
+            classOf[Object {...}].getDeclaredField("x$lzy1")
+          )
+        ()
+      }
+    @static @static val OFFSET$_m_0: Long = 
+      scala.runtime.LazyVals.getStaticFieldOffset(
+        classOf[Object {...}].getDeclaredField("x$lzy1")
+      )
+    private def writeReplace(): Object = 
+      new scala.runtime.ModuleSerializationProxy(classOf[A])
+    @volatile lazy <static> var x$lzy1: Object = null
+    lazy def x(): Int = 
+      {
+        var result: Object = A#x$lzy1
+        if result.isInstanceOf[Int] then return scala.Int.unbox(result) else 
+          if result.==(scala.runtime.LazyVals.NullValue) then 
+            return scala.Int.unbox(null)
+           else return scala.Int.unbox(A.x$lzyINIT1())
+      }
+    private def x$lzyINIT1(): Object = 
+      while <empty> do 
+        {
+          val current: Object = A#x$lzy1
+          if current.==(null) then 
+            if 
+              scala.runtime.LazyVals.objCAS(classOf[A], A.OFFSET$_m_0, null, 
+                scala.runtime.LazyVals.Evaluating
+              )
+             then 
+              {
+                var result: Object = null
+                var resultNullable: Object = null
+                try 
+                  {
+                    resultNullable = scala.Int.box(2)
+                    if resultNullable.==(null) then 
+                      result = scala.runtime.LazyVals.NullValue
+                     else result = resultNullable
+                    return resultNullable
```

Comment:
Putting this `return resultNullable` *after* the `try..finally` may produce better bytecode. `return`s from the body of a `try` cause minutiae in the compiler back-end to be correctly handled.